### PR TITLE
batches: change chevrons from up/down to down/right

### PR DESCRIPTION
**Pull request by mrnugget**
See [this Slack thread](https://sourcegraph.slack.com/archives/C03CSAER9LK/p1691502296112349) in which Daniel and Rob agree that we should be consistent and that the preferred variant is the one introduced in #54911.

## Test plan

- Storybook

See:

screenshot_2023-08-10_13 03 17@2x
screenshot_2023-08-10_13 03 36@2x

### CODE REVIEW

**Review by eseliger — APPROVED** [review]

**Review by BolajiOlajide — APPROVED** [review]

**Comment by mrnugget** [comment]
> And maybe the only thing worse than a confusing UI is a confusing _and inconsistent_ UI. 😂

It _is_ already inconsistent, right? Because it has been changed in the reference panel, after design decided that the previous approach was wrong.

Can we do a bit-by-bit migration or do you think we should update all of them?

**Comment by danielmarquespt** [comment]
Ok, rule of thumb for this:

▶️ 🔽
Should be used for expandable sections **that push the content**. File trees are a good example.
image

🔽🔼 
Should be used for menus that **overlays a list of options** over the content.

image


Now, the problem is that we sometimes for **expanding sections**, we put the icon on the right:

Screenshot 2023-08-17 at 14 35 20

Which is correct for Overlays, but its not for expanding section.

Why? Because if we follow our rule of "pushing content means: ▶️ 🔽" and we put that on the right, then we end up with something like this:

Screenshot 2023-08-17 at 14 38 07

which is what is used to navigate to submenus (specially on mobile), and may cause some confusion
Screenshot 2023-08-17 at 14 39 16

So, in order to fix this, we can't just replace everything, because the place is also wrong.

**TL;DR:**

Screenshot 2023-08-17 at 14 55 21

**Comment by danielmarquespt** [comment]
Disclaimer, some apps are not strict about this, Slack **on mobile** is displaying right arrows, for content that expands.

Personally i'm not a fan, I thing " > " on the right is a strong signifier of "dive in / navigate to" not "show more"  

Screenshot 2023-08-17 at 14 39 16


Whatever we pick, we need to be consistent.
